src/api/app/controllers/webui/project_controller.rb

```diff
@@ -314,12 +314,6 @@ def update_target
   end
 
   def repositories
-    unless @project
-      # TODO support flagdetails for remote instances in the API
-      flash[:error] = "You can't show repositories for remote instances"
-      redirect_to :action => :show, :project => params[:project]
-      return
-    end
```

Comment:
I will something like "Cleanup of dead code." because this action is behind a before filter that raises an exception if no project is set properly... so never executed. Is ok for you?